packages/p2p-v2/src/pages/my-profile/screens/MyProfileDailyLimit/MyProfileDailyLimit.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from 'react';
+import { DailyLimitModal } from '@/components/Modals';
+import { useAdvertiserStats, useDevice } from '@/hooks';
```

Comment:
```suggestion
import { DailyLimitModal } from '@/components';
```

Reply:
actually we can keep it like `@/components/Modals`, makes it less likely to develop circular dependency